apps/ledger-live-mobile/src/dynamicContent/useContentCards.ts

```diff
@@ -1,15 +1,19 @@
-import { useEffect } from "react";
+import { useCallback, useEffect } from "react";
 import { useDynamicContentLogic } from "./useDynamicContentLogic";
 
 const HookDynamicContentCards = () => {
   const { refreshDynamicContent, fetchData, clearOldDismissedContentCards } =
     useDynamicContentLogic();
 
-  useEffect(() => {
+  const fetchContentCardsData = useCallback(async () => {
```

Comment:
Do we still need the changes in this file ?

Reply:
Yes otherwise it closes the LP after loads